tests/websockets/test_graphql_transport_ws.py

```diff
@@ -156,6 +161,49 @@ async def test_connection_init_timeout_cancellation(
         assert ws.closed
 
 
+async def test_close_twice(
+    mocker: MockerFixture, request, http_client_class: Type[HttpClient]
+):
+    if sys.version_info < (3, 8):
+        pytest.skip("Task name was introduced in 3.8 and we need it for this test")
```

Comment:
can we move this to a marker on the test? 

```python
@pytest.mark.skipif(...)
async def test_close_twice(
```

Reply:
Sure! Will do that